tests/integration/lsp_tests.rs

```diff
@@ -80,7 +80,55 @@ fn lsp_tsconfig_types() {
 
   let mut client = context.new_lsp_command().build();
   client.initialize(|builder| {
-    builder.set_config("types.tsconfig.json");
+    builder
+      .set_config("types.tsconfig.json")
+      // avoid finding the declaration file via the document preload
+      .set_preload_limit(0);
+  });
+
+  let diagnostics = client.did_open(json!({
+    "textDocument": {
+      "uri": Url::from_file_path(temp_dir.path().join("test.ts")).unwrap(),
```

Comment:
```suggestion
      "uri": temp_dir.uri().join("test.ts").unwrap(),
```

Reply:
I went through and updated all these instances in this file.